core/src/main/java/com/amplifyframework/api/graphql/Page.java

```diff
@@ -0,0 +1,9 @@
+package com.amplifyframework.api.graphql;
+
+public abstract class Page<M> {
```

Comment:
I'm open to suggestions on other names for this class.  We actually already have a `Page` object defined in core, which is used for local pagination via DataStore, so it's kinda confusing to have this one with the same name.

Reply:
Does `PaginatedResult` reflect the intent? (or something along those lines)

Reply:
I like that!  I'll update it.